app/controllers/application_controller.rb

```diff
@@ -127,6 +128,10 @@ def authority_forbidden(error)
       render text: I18n.t('not_authorized'), layout: true, status: :forbidden
     end
 
+    rescue_from 'EmailConnectionError' do |e|
```

Comment:
Unused block argument - `e`. You can omit the argument if you don't care about it.<br>Inconsistent indentation detected.